routes.jsx

```diff
@@ -15,21 +15,38 @@ import Navbar from './components/navbar/navbar.jsx';
 import Footer from './components/footer/footer.jsx';
 
 const App = React.createClass({
+  getInitialState() {
+    return {
+      suppressSplashScreen: false
+    };
+  },
   componentDidMount() {
-    setTimeout(function() {
-      this.splash.classList.add(`dismissed`);
-      document.querySelector(`#app`).classList.add(`splash-dismissed`);
-    }, 3000);
+    const suppressSplashScreenKey = `suppressSplashScreen`;
+
+    this.setState({
```

Comment:
i.e., create a lib file that has the shim code, and instead of just doing `localStorage.method()` I do `require('storage').method()`